apps/admin-gui-e2e/src/integration/group-management-perun_admin.spec.js

```diff
@@ -0,0 +1,223 @@
+        .type('test-group-description')
+      .get('[data-cy=create-group-button-dialog]')
+        .click()
+      .wait('@createGroup')
+      // assert that group was created
+      .get('[data-cy=test-group-checkbox]')
+      .should('exist')
+  });
+
+  it('test remove subgroup', () => {
+
+    cy.intercept('**/groupsManager/deleteGroups').as('deleteGroups')
+      .get('[data-cy=subgroups]')
+      .click()
+      .get(`[data-cy=${dbSubGroupName}-checkbox]`)
+      .click()
+      .get('[data-cy=delete-group-button]')
+      .click()
+      .get('[data-cy=delete-button-dialog]')
+      .click()
+      .wait('@deleteGroups')
+      // assert that group was deleted
+      .get(`[data-cy=${dbSubGroupName}-checkbox]`)
+      .should('not.exist')
+  });
+
+  it('test add attribute', () => {
+
+    cy.intercept('**/attributesManager/setAttributes/**').as('setAttributes')
+      .get('[data-cy=attributes]')
+        .click()
+      .get('[data-cy=add-attributes]')
+        .click()
+      .get('[data-cy=filter-attributes]')
+        .type('footer')
+      .get(`[data-cy=${addedAttribute}-value]`)
+        .type('test')
+      .get('[data-cy=save-selected-attributes]')
+        .click()
+      .wait('@setAttributes')
+      // assert that attribute exists
+      .get(`[data-cy=${addedAttribute}-value]`)
+      .should('exist')
+  });
+
+  it('test delete attribute', () => {
+
+    cy.intercept('**/attributesManager/removeAttributes/**').as('removeAttributes')
+      .get('[data-cy=attributes]')
+        .click()
+      .get(`[data-cy=${dbGroupAttribute}-checkbox]`)
+        .click()
+      .get('[data-cy=remove-attributes]')
+        .click()
+      .get('[data-cy=delete-attributes]')
+        .click()
+      .wait('@removeAttributes')
+      // assert that attribute exists
+      .get(`[data-cy=${dbGroupAttribute}-checkbox]`)
+      .should('not.exist')
+  });
+
+  it('test add group member', () => {
+
+    cy.intercept('**/groupsManager/addMember**').as('addMember')
+      .get('[data-cy=members]')
+        .click()
+      .get('[data-cy=add-members]')
+        .click()
+      .get('[data-cy=search-members]')
+        .type(`${dbUser}`)
+      .get('[data-cy=search-button]')
+        .click()
+      .get(`[data-cy=${dbUser}-checkbox]`)
+        .click()
+      .get('[data-cy=add-button]')
+        .click()
+      .wait('@addMember')
+      // assert that member was created
+      .get(`[data-cy=${dbUser}-checkbox]`)
+      .should('exist')
+  });
+
+  it('test remove group member', () => {
+
+    cy.intercept('**/groupsManager/removeMember**').as('removeMember')
+      .get('[data-cy=members]')
+        .click()
+      .get(`[data-cy=${dbUser}-checkbox]`)
+        .click()
+      .get('[data-cy=remove-members]')
+        .click()
+      .get('[data-cy=remove-members-dialog]')
+        .click()
+      .wait('@removeMember')
+      // assert that member was removed
+      .get(`[data-cy=${dbUser}-checkbox]`)
+      .should('not.exist')
+  });
```

Comment:
You forgot to rewrite also this test (${dbUser} is in add test and also in remove test). Please make this test independent too. You probably should make some changes also in the GitLab repo.